trinity/extensibility/plugin_manager.py

```diff
@@ -74,7 +107,53 @@ def broadcast(self, event: BaseEvent, exclude: BasePlugin = None) -> None:
             if not plugin.should_start():
                 continue
 
-            plugin.start(None)
+            context = self._create_context_for_plugin(plugin)
+            plugin.start(context)
             self._started_plugins.append(plugin)
             self._logger.info("Plugin started: {}".format(plugin.name))
             self.broadcast(PluginStartedEvent(plugin), plugin)
+
+    # FIXME: This is a temporary API
+    def start(self, boot_kwargs: Dict[str, Any]) -> None:
+        for plugin in self._plugin_store:
+
+            # This whole API is going away so for now, we aren't caring about
+            # any of the other plugins
+
+            if plugin.process_scope is not PluginProcessScope.OWN:
+                continue
+
+            context = self._create_context_for_plugin(plugin, boot_kwargs)
+            plugin.start(context)
+
+    def _is_responsible_for_plugin(self, plugin: BasePlugin) -> bool:
+
+        main_and_own_scopes = [PluginProcessScope.MAIN, PluginProcessScope.OWN]
```

Comment:
You should use `set` notation when creating a data structure for *inclusion* checks.

```python
{PluginProcessScope.Main, PluginProcessScope.OWN}
```

Also, maybe store this as a top level module variable so we don't incure the minor but additional runtime overhead of constructing this set inline.